tasks/set_vars.yml

```diff
@@ -4,7 +4,28 @@
 - name: Lookup applications and set the apps var
   set_fact: apps="{{ lookup('pipe', 'python plugins/utils/lookup_apps.py') | from_yaml }}"
 
-- name: Display looked up applications
+# If the playbook is invoqued with the app_filter extra var, we restrict apps
+# to the list passed as an argument, e.g.:
+#
+#     $ ansible-playbook foo.yml -e "app_filter='richie,redirect'"
+#
+# This task will build the jmespath query to restrict apps to the selected
+# one(s), e.g.:
+#
+#    [?name=='richie' || name=='redirect']
```

Comment:
For me redirect, like openshift-acme, must be present in all project.
Is not a good exemple, IMHO.